src/vm/method.cpp

```diff
@@ -2764,7 +2764,7 @@ void MethodDesc::Save(DataImage *image)
 
         if (pNewSMD->HasStoredMethodSig())
         {
-            if (!image->IsStored((void *) pNewSMD->m_pSig))
+            if (!image->IsStored((void *) pNewSMD->m_pSig.GetValueMaybeNullEmbedded(DPTR_FOR_POINTER(pNewSMD))))
```

Comment:
This is not DACized code. You call the simpler non-DACized `GetValue*` overloads here.